eng/templates/Azure.ResourceManager.Template/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## 1.0.0-beta.1 (Unreleased)
 
+### General New Features
+
+    - Support MSAL.NET, Azure.Identity is out of box for supporting MSAL.NET
+    - Support [OpenTelemetry](https://opentelemetry.io/) for distributed tracing
+    - HTTP pipeline with custom policies
+    - Better error-handling
```

Comment:
```suggestion
    - Improve error handling
```

Reply:
We already have same description in the [changelog](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resources/Azure.ResourceManager.Resources/CHANGELOG.md#general-new-features) of `Azure.ResourceManager.Resources`, so I prefer to keep it as it is.